store/store.go

```diff
@@ -531,11 +531,55 @@ func (s *SnapUbuntuStoreRepository) FindSnaps(searchTerm string, channel string,
 	return snaps, nil
 }
 
+// CurrentSnap contains information for the store about the currently
+// installed snap so that the store can decide what update we should see
+type CurrentSnap struct {
+	SnapID      string
+	Revision    snap.Revision
+	Epoch       string
+	Confinement snap.ConfinementType
```

Comment:
Should probably be DevMode, per earlier comment, and reflect the current state rather than the snap opinion.